### DISCUSSION TITLE
No changes since 0.4.5?

### DISCUSSION THREAD

**Discussion by brysonreece** [IceWhaleTech/CasaOS#1837] [General]
Can anyone explain why the package has continuously been bumped, beginning with 0.4.6, to the latest release with no actual changes being made to the codebase? Are there hashes available we can compare release to, in order to ensure undocumented changes aren't being added to release assets?

**Comment by dragonfire1119** [comment]
Changes are here:

https://github.com/IceWhaleTech/CasaOS-UI

https://github.com/IceWhaleTech/CasaOS-AppManagement

https://github.com/IceWhaleTech/CasaOS-App-UI
